encord_agents/gcp/dependencies.py

```diff
@@ -106,11 +107,46 @@ def my_agent(
     return np.asarray(img, dtype=np.uint8)
 
 
+def dep_asset(lr: LabelRowV2) -> Generator[Path, None, None]:
```

Comment:
Seems odd that we have to duplicate dependencies for FastAPI vs GCP. Could this be centralised

Reply:
No it cannot - unfortunately. FastAPI use their Depends and resolve logic, we use ours.